oras/provider.py

```diff
@@ -747,7 +747,12 @@ def push(self, *args, **kwargs) -> requests.Response:
 
         # Config is just another layer blob!
         logger.debug(f"Preparing config {conf}")
+        if config_file is None:
+            config_file = oras.utils.get_tmpfile(suffix=".json")
+            with open(config_file, "w") as f:
```

Comment:
You can use oras.utils.write_file here: https://github.com/oras-project/oras-py/blob/02c5bdacb54b5d7d6c152fcdbc0e11cf69b4900f/oras/utils/fileio.py#L235 should just need to do:

```python
oras.utils.write_file(config_file, "{}")
```

Reply:
And I guess we are OK here not cleaning up?

Reply:
* Fixing the use of write_file
* Delete temporary files

Two corrections were made.